src/Features/Core/Portable/SolutionCrawler/SolutionCrawlerService.cs

```diff
@@ -7,11 +7,17 @@
 using System.Composition;
 using Microsoft.CodeAnalysis.Host;
 using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Options;
 
 namespace Microsoft.CodeAnalysis.SolutionCrawler
 {
     internal partial class SolutionCrawlerRegistrationService : ISolutionCrawlerRegistrationService
     {
+        // TODO:
```

Comment:
What is there todo with the EnableSolutionCrawler option?

Reply:
Not needed anymore. Removed.